golang/pkg/dagent/config/secret.go

```diff
@@ -4,11 +4,13 @@ import (
 	"fmt"
 
 	config "github.com/dyrector-io/dyrectorio/golang/internal/config"
+	"github.com/rs/zerolog/log"
 )
 
 type ConfigFromFile string
 
 func (field *ConfigFromFile) SetValue(secretPath string) error {
+	log.Debug().Msgf("loading config from file: %v", secretPath)
```

Comment:
We are using `Str(key, value)` and `Msg()` for printing out variables. (I just want to point out)

Reply:
Done, thx.